Add SMP missing release notes (#5280)
harness/developer-hub: release-notes/continuous-delivery.md

- Deployment failing with Terraform error on infra provisioners. (CDS-85684) Terraform tasks working directory was created based on account, org, project, and provisioner identifier.
  - This combination might cause issues if two steps with same account, org, project, and provisioner identifier are running simultaneously on same delegate.
  - With this change every Terraform step execution will run in a unique working directory.
- Console logs missing from CloudFormation steps. There was an issue where CloudFormation steps were not updating the console longs when there are multiple steps with a similar prefix. This issue is now fixed.(CDS-84962, ZD-53810, ZD-53865)
- When internal looping strategy fields marked as runtime, the run pipeline form did not prompt for input. Condition updated to validate looping strategy runtime inputs. (CDS-86445)
- Blank tile on ASG traffic shifter. Blank tile on the UI for the ASG Traffic Shifter fixed. (CDS-86298)

> - Customer unable to set environment infrastructure definition. Infrastructure definition now set with provisioner step group template. (CDS-86109, ZD-54775)

Reviewer: ```suggestion
- Infrastructure definitions didn't allow you to set an environment. (CDS-86109, ZD-54775)

   Infrastructure definitions now allow you to set the provisioner step group template.
```

> - Unsaved Changes Doesn't Display Whitespace Changes. Whitespace difference consideration implemented in diff editors. (CDS-85964)

Reviewer: ```suggestion
- Diff editors didn't display unsaved whitespace changes. (CDS-85964)

   Whitespace difference consideration has been implemented in diff editors to resolve this issue.
```
